recipe/meta.yaml

```diff
@@ -1,12 +1,11 @@
 {% set version = "1.81.0" %}
-{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
 
 package:
   name: libboost-mpi
   version: {{ version }}
 
 source:
-  url: https://boostorg.jfrog.io/artifactory/main/release/{{ version }}/source/{{ filename }}
+  url: https://boostorg.jfrog.io/artifactory/main/release/{{ version }}/source/boost_{{ version|replace(".", "_") }}.tar.bz2
```

Comment:
Pulled in this same change from PR ( https://github.com/conda-forge/boost-cpp-feedstock/pull/133 )